boards/arm/thingy53_nrf5340/Kconfig

```diff
@@ -59,6 +59,52 @@ config DOMAIN_CPUNET_BOARD
 	  another board. For example hci_rpmsg on the nRF5340_cpunet for
 	  Bluetooth applications.
 
+config BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+	bool "MCUmgr and OTA DFU"
+	select MCUMGR
+	select MCUMGR_SMP_BT
+	select MCUMGR_CMD_IMG_MGMT
+	select MCUMGR_CMD_OS_MGMT
+	select MCUMGR_GRP_ZEPHYR_BASIC
+	select MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE
+	select OS_MGMT_MCUMGR_PARAMS
+	select NORDIC_QSPI_NOR
+	select MCUMGR_SMP_REASSEMBLY_BT
+	help
+	  Enable this option to include MCUmgr in the build which will enable
+	  the Bluetooth transport and image management group, which allows
+	  for firmware over the air update support of the device.
+
+config MCUMGR_SMP_BT_AUTHEN
+	default n if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+
+config UPDATEABLE_IMAGE_NUMBER
+	default 2 if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+
+config BT_L2CAP_TX_MTU
+	default 498 if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+
+config BT_BUF_ACL_RX_SIZE
+	default 502 if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+
+config BT_BUF_ACL_TX_SIZE
+	default 502 if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+
+config BT_CTLR_DATA_LENGTH_MAX
```

Comment:
Is it okay to configure the Bluetooth Controller configuration in the Application image? I think this could produce a warning during the build time as the Bluetooth Controller configuration is not available in the Application image of the nRF5340

Reply:
I don't think that this particular option is available in the application image as the Bluetooth Controller configuration should be disabled there. It is not present in the autoconf.h file from the SMP server sample (built for nRF5340 DK):

```cat build//zephyr/include/generated/autoconf.h | grep 'BT_CTLR'```

It may be the case that this configuration option does not have any impact on the building stage, but it seems unnecessary

Reply:
Moved